src/pages/tools/mutationMapper/MutationMapperToolStore.ts

```diff
@@ -171,7 +171,7 @@ export default class MutationMapperToolStore
                         this.canonicalTranscriptsByHugoSymbol.result[gene.hugoGeneSymbol].transcriptId : undefined;
                     
                     const getMutations = () => {
-                        if (canonicalTranscriptId && this.indexedVariantAnnotations.result) {
+                        if (canonicalTranscriptId && this.indexedVariantAnnotations.result && !_.isEmpty(this.indexedVariantAnnotations.result)) {
```

Comment:
it might be nice to use some flag that the input annotations are protein changes - but this works too i guess